doc/downloading.md

```diff
@@ -0,0 +1,27 @@
+# Downloading SPIRE
+
+This page describes the options available for downloading SPIRE.
+
+## Docker Compose
```

Comment:
The SPIRE 101 walkthrough is outdated (I filed #1815 to update it) and we don't have a docker-compose file anymore. In any case I think that the SPIRE 101 walkthrough relies on cloning the SPIRE repository first as the primary download method. I don't think that we should have a `Docker Compose` section here.

Reply:
Hi Agustín - See my comments in 303552e and let me know what you think.